clusterloader2/pkg/measurement/common/probes/probes.go

```diff
@@ -173,6 +173,13 @@ func (p *probesMeasurement) start(config *measurement.Config) error {
 		return err
 	}
 	if err := p.waitForProbesReady(config); err != nil {
+		if err.Error() == wait.ErrWaitTimeout.Error() {
+			// Start the measurement nonetheless as timing out during waiting for
+			// probe pods to be ready is almost always because of a small fraction
```

Comment:
Let's not talk abotu "small fraction" - in small clusters there is one probe and then if it's not ready it's 100% of them.

Let's only say that that this usually means that not all probes are visible via promethes.

Reply:
it is at least 3 pods for currently supported 100 nodes. 2 nodes for small clusters (supported in the future)

Reply:
Still - at least 1 has to be down, and 1 out of 3 is not a "small fraction".

Reply:
lets skip "small", just a fraction